ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -350,6 +350,18 @@ def populate_fail_info(self, fails):
                 self.fails[key] = set()
             self.fails[key] |= fails[key]
 
+    def get_preboot_info(self):
+        msg = 'Preboot oper: %s ' % self.preboot_oper
+        if ':' in self.preboot_oper:
+            temp = self.preboot_oper.split(':')
```

Comment:
`temp` is not the best name in this case. I'd better use someting more meaningful

Reply:
renamed